build.make

```diff
@@ -62,6 +62,12 @@ endif
 
 ARCH := $(if $(GOARCH),$(GOARCH),$(shell go env GOARCH))
 
+# Specific BASE_IMAGE for different arch, default BASE_IMAGE gcr.io/distroless is for x86, discolix/static is for arm.
+BASE_IMAGE=gcr.io/distroless/static:latest
+ifeq (${ARCH}, arm64)
+  BASE_IMAGE=discolix/static:latest
```

Comment:
This image doesn't seem very widely used. I am checking to see if the Kubernetes project has another recommended base image for arm

Reply:
Thanks, @msau42. I found it in this repo https://github.com/discolix/discolix.